docs/UserGuide.adoc

```diff
@@ -207,41 +207,86 @@ exp/23-Nov-2020 a/21`
 
 ==== Add with system prompting
 
-In this mode, the user can add the person's attributes one by one.
-After entering an attribute, the system will prompt the user input for another attribute.
+In this mode, instead of typing a long command, you can add a person's attributes one by one.
+After entering an attribute, the system will prompt you to input another attribute.
 
-Format: `add t/**TYPE**`
+To add a person in this mode, enter the following command of the format: `add t/**TYPE**`
 
-When the user type in the command above, a form will be displayed. For example, if a user wants to add a patient,
-type `add t/patient` to the command box. The form will be as below:
+ORGANice will prompt you to type in each attribute, one by one until all the fields are filled.
+Here are some special commands that might be useful when you are in the process of filling up the form:
 
-image::form.png[width="790"]
+* `/abort` - The command to abandon the current form that you are currently filling.
+* `/undo` - The command to undo the last entry that you entered, this can be useful as sometimes you might make some careless mistakes.
+* `/done` - The command to indicate that you are done with filling all the required attributes and want to add the person into the application.
 
-ORGANice will prompt the user to type in each attribute, one by one until all the fields are filled.
+Note that both `/abort` and `/undo` commands can be used at any point in times, while `/done` command can only be used once all the required attributes have been specified.
 
-For example, when a user wants to add a patient, the display box will prompt the user to enter the patient's name.
-For the user to add a patient that is named "John Doe", type `John Doe` and then press enter.
-After entering the name, the patient's name will be displayed in the form, as shown below.
+That's the important commands for this feature, now let's see an example on how the feature will look like.
```

Comment:
**These** are the important commands for this feature, **so** now let's **take a look at an example of this how this feature can be used.**